tests/container_test.py

```diff
@@ -42,17 +40,9 @@ def test_wait_for_ready(main_container):
 
 def test_wait_for_exits(main_container, version_container):
     """Wait for containers to exit."""
-    assert main_container.wait() == 0, "Container service (main) did not exit cleanly"
-    assert (
-        version_container.wait() == 0
-    ), "Container service (version) did not exit cleanly"
-
-
-def test_output(main_container):
-    """Verify the container had the correct output."""
-    main_container.wait()  # make sure container exited if running test isolated
-    log_output = main_container.logs().decode("utf-8")
-    assert SECRET_QUOTE in log_output, "Secret not found in log output."
+    main_container.stop()
+    assert main_container.wait() == 143, "Container service (main) exited"
+    assert version_container.wait() == 0, "Container service (version) exited"
```

Comment:
The messages here are meant to state what happened if the assertion was not met - I believe you want messages like this, but please confirm:
```suggestion
    assert main_container.wait() == 143, "Container service (main) did not stop cleanly"
    assert version_container.wait() == 0, "Container service (version) did not exit cleanly"
```

Reply:
@dav3r that makes sense. resolved with [4b1d0a2](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/4b1d0a2beb4f09892a44d16eda557ace2b042b3f)